experiments/functional/zfs-LocalPV/zfspv-raw-block-volume/test.yml

```diff
@@ -0,0 +1,170 @@
+- hosts: localhost
+  connection: local
+  gather_facts: False
+
+  vars_files:
+    - test_vars.yml
+    
+  tasks:
+    - block:
+
+        ## Generating the testname for deployment
+        - include_tasks: /utils/fcm/create_testname.yml
+
+        ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: "/utils/fcm/update_litmus_result_resource.yml"
+          vars:
+            status: 'SOT'
+        
+        - name: Create the namespace for block-volume of zfspv
+          shell: >
+            kubectl create ns {{ app_ns }}
+          args:
+            executable: /bin/bash
+
```

Comment:
where do we create storage class with raw block mode

Reply:
we are creating this storage class in zfspv-provisioner.